R/install.R

```diff
@@ -126,8 +126,12 @@ renv_install_impl <- function(record, project) {
   # figure out whether we can use the cache during install
   library <- renv_global_get("install.library") %||% renv_libpaths_default()
   linkable <-
-    settings$use.cache(project = project) &&
-    identical(library, renv_paths_library(project = project))
+    settings$use.cache(project = project) && (
+      identical(library, renv_paths_library(project = project)) ||
+        # Support for packaged renv under inst
+        # Remove inst sub-folder from path
+      identical(sub("inst/","",library), renv_paths_library(project = project))
```

Comment:
Can you explain the rationale here?

Reply:
This change belongs to another branch, I have moved it.
see this PR https://github.com/rstudio/renv/pull/166